config/replication_exclude_tables.yml

```diff
@@ -85,7 +85,6 @@
 - server_roles
 - sessions
 - vim_performance_states
-- vim_performance_tag_values
```

Comment:
Why is this removed?  Also, does this file even matter anymore?

Reply:
I think this will make more sense if I answer these in the reverse order ...

As of the commit that changes this line this file is *the* list of excluded tables. So every time we configure the publication we will read this file (and only this file) to determine what should be included.

As for your first question ... Because of the importance of this file I decided to change the spec that checks the included files to ensure that this file is consistent with the current set of tables in the database. This means the spec will fail if there are any tables in this file which don't exist in the current schema. That was the case for this table which was removed in https://github.com/ManageIQ/manageiq-schema/commit/4bf30fb